### ISSUE TITLE
Currently Not able to use special characters in Request tests tab and its displaying error

### ISSUE THREAD

**Issue by vishnuprakash9845** [a85/POSTMan-Chrome-Extension#11081]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

I am getting the response like this.

![image](https://user-images.githubusercontent.com/24207575/178678138-eef5da3c-4598-4178-89c1-2edd86f63849.png)

I want the value underlined and set to global variable. I can use below json path to get it.

![image](https://user-images.githubusercontent.com/24207575/178678207-636c78a4-cacb-41a5-bdce-d1db7e6882dc.png)

But in postman If I try same json path, its giving error. Any way to avoid it or any workaround is there.

![image](https://user-images.githubusercontent.com/24207575/178678273-75ad9b85-62e0-4cd1-8845-1d8f7ccce534.png)

I tried googling and approached Manual API Testers to find the solution for this issue, hard luck couldn't get it.

### Steps To Reproduce

1. Open the Postman Application
2. Perform POST request
3. Form the response the try to get the value using wild card (*)

### Screenshots or Videos

Attached in the description

### Operating System

Windows

### Postman Version

9.24.0

### Postman Platform

Postman App

**Comment by DannyDainton** [comment]
Hey @vishnuprakash9845 

`JSONpath` isn't part of the Postman sandbox environment so that's a valid error in this context. 

You would need to directly access the `value` by specifically adding the `key` to the reference path, using bracket notation. 

```javascript
pm.environment.set("nameBoxElement", response.value['element-<the guid>'])
```

Or you could make that a bit more dynamic using this, which would grab the value from the first key in the `value` object.

```javascript
pm.environment.set("nameBoxElement", _.values(response.value)[0])
```

I'm going to close this issue as it's not a problem related to the Postman platform and more about the way you would reference different parts of a JSON response.

If you have any further questions like this, the best place to ask them is on our [Community forum](https://community.postman.com/).

**Comment by vishnuprakash9845** [comment]
Hi @DannyDainton 

Above approach will fail in this scenario.

The guid is dynamic sometimes so can't hardcode in the json path,  need construct the path with static text 'element-' and some wild card characters. And finally store in environment variable. 

![image](https://user-images.githubusercontent.com/24207575/178737676-9ee6aee4-60a5-4d18-93e1-4c59799d5136.png)

Even I tried the second approach, it also didn't work.

Updated in Tests
![image](https://user-images.githubusercontent.com/24207575/178739074-11b0413c-6a3c-4688-92eb-b8de78145271.png)


In Variables updated as
![image](https://user-images.githubusercontent.com/24207575/178739166-16c15b75-ef42-4598-8578-d22f7dd56f6e.png)
